wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -517,8 +517,6 @@ public final class ZMUserSession: NSObject {
             self.applicationStatusDirectory.clientRegistrationStatus.determineInitialRegistrationStatus()
             self.hasCompletedInitialSync = self.applicationStatusDirectory.syncStatus.isSlowSyncing == false
 
-            createMLSClientIfNeeded()
```

Comment:
Is the `func createMLSClientIfNeeded()` still needed or can it be delete following this line?

Reply:
Can be deleted!